docs/source/using-discovery/workingwithgpu.rst

```diff
@@ -139,78 +181,127 @@ There are several versions of CUDA Toolkits on Discovery, as listed below.::
 Using GPUs with TensorFlow
 ==========================
-We recommend that you use CUDA 11.2 (latest supported version) when working on a GPU with the latest version of TensorFlow (TF).
-You can find the compatibility of CUDA and TensorFlow versions at the following website https://www.tensorflow.org/install/source#gpu and for detailed installation instructions also visit https://www.tensorflow.org/install/pip.
-
-For the latest installation, use the TensorFlow pip package which includes GPU support for CUDA-enabled devices::
-
-  srun --partition=gpu --nodes=1 --pty --gres=gpu:1 --ntasks=1 --mem=4GB --time=01:00:00 /bin/bash
-  module load anaconda3/2022.05
-  module load cuda/11.2
-  conda create --name TF_env python=3.9 -y #where TF_env is the name of the conda environment
-  source activate TF_env #load the virtual conda environment "TF_env"
-  export LD_LIBRARY_PATH=$HOME/.conda/envs/TF_env/lib:$LD_LIBRARY_PATH
-  conda install -c conda-forge cudatoolkit=11.2 cudnn=8.1.0 -y 
-  pip install tensorflow
+We recommend that you use CUDA 11.2 (latest supported version) when
+working on a GPU with the latest version of TensorFlow (TF).
+TensorFlow provides information on the `compatibility of CUDA and
+TensorFlow versions <https://www.tensorflow.org/install/source#gpu>`_,
+and `detailed installation instructions
+<https://www.tensorflow.org/install/pip>`_.
+
+For the latest installation, use the TensorFlow pip package, which
+includes GPU support for CUDA-enabled devices::
+
+  srun --partition=gpu --gres=gpu:1 --nodes=1 --cpus-per-task=2 --mem=10GB --time=02:00:00 --pty /bin/bash
+  module load anaconda3/2022.05 cuda/11.2
+  conda create --name TF_env python=3.9 -y
+  source activate TF_env
+  conda install -c conda-forge cudatoolkit=11.2.2 cudnn=8.1.0 -y
+  export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$CONDA_PREFIX/lib/
+  mkdir -p $CONDA_PREFIX/etc/conda/activate.d
+  echo 'export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$CONDA_PREFIX/lib/' > $CONDA_PREFIX/etc/conda/activate.d/env_vars.sh
+  pip install --upgrade pip
+  pip install tensorflow==2.11.*
 
 Verify the installation::
 
   # Verify the CPU setup (if successful, then a tensor is returned):
   python3 -c "import tensorflow as tf; print(tf.reduce_sum(tf.random.normal([1000, 1000])))"
-  # verify the GPU setup (if successful, then a list of GPU devices is returned):
+
+  # verify the GPU setup (if successful, then a list of GPU device is returned):
   python3 -c "import tensorflow as tf; print(tf.config.list_physical_devices('GPU'))"
+
   # test if a GPU device is detected with TF (if successful, then True is returned):
   python3 -c 'import tensorflow as tf; print(tf.test.is_built_with_cuda())' 
 
 To get the name of the GPU, type::
 
    python -c 'import tensorflow as tf;  print(tf.test.gpu_device_name())'
 
-If the installation is successful, then, for example, you should see the following output::
+If the installation is successful, then, for example, you should see
+the following as an output,::
```

Comment:
Should we say a similar output like this? This can vary slightly based on the gpu the user lands on since it wasn't specified in the steps above.

Reply:
That's why I had put "for example" but if you think "similar output" would be better, then I can change to that as well @joseph-cooke

Reply:
Looking back over it, for example should be sufficient. I just wasn't sure if users would look at the output and raise a ticket if they saw something slightly differently.